src/components/Badge/Badge.js

```diff
@@ -26,7 +26,8 @@ const DropItem = styled(Button)`
 `
 
 const DropLink = styled(Link)`
```

Comment:
`DropLink` looks fine, but what about `StyledBox`, just below this code block?

```
const StyledBox = styled(Box)`
  background: ${props => props.disabled ? '#D8D8D8' : 'inherit'}
  :hover {
    background: #D8D8D8;
  }
```

Reply:
Oops! I missed that one

Reply:
Looks like that came in when this branch was rebased. Good catch.